Ginger/GingerCore/GeneralLib/General.cs

```diff
@@ -1319,6 +1320,16 @@ public static bool IsAdmin()
             return false;
 
         }
+
+        public static ObservableList<SolutionCategoryValue> GetSolutionReleaseValues()
```

Comment:
i don't General class the proper place for this, need to add it to Workspace.cs or to SolutionOperations classes

Reply:
moved in SolutionOperations  class